packages/wonder-blocks-popover/src/components/__docs__/popover-content.argtypes.js

```diff
@@ -0,0 +1,116 @@
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {LabelLarge} from "@khanacademy/wonder-blocks-typography";
+
+const ActionsMappings: {[key: string]: React.Node} = {
+    singleButton: <Button kind="primary">Continue</Button>,
+    singleButtonEmphasized: (
+        <Button kind="primary" light={true}>
+            Continue
+        </Button>
+    ),
+    pagination: (
+        <>
+            <Button kind="tertiary">Previous</Button>
+            <Strut size={Spacing.medium_16} />
+            <Button kind="primary">Next</Button>
+        </>
+    ),
+    paginationEmphasized: (
+        <>
+            <Button kind="tertiary" light={true}>
+                Previous
+            </Button>
+            <Strut size={Spacing.medium_16} />
+            <Button kind="primary" light={true}>
+                Next
+            </Button>
+        </>
+    ),
+    steps: (
+        <>
+            <LabelLarge>Step 1 of 5</LabelLarge>
+            <Spring />
+            <Button kind="tertiary">Skip this step</Button>
+        </>
+    ),
+    stepsEmphasized: (
+        <>
+            <LabelLarge>Step 1 of 5</LabelLarge>
+            <Spring />
+            <Button kind="secondary" light={true}>
+                Skip this step
+            </Button>
+        </>
+    ),
+};
+
+export default {
+    content: {
+        description: "The content to render inside the popover.",
+        type: {required: true, summary: "string"},
+    },
+    title: {
+        description: "The popover title",
+        type: {required: true, summary: "string"},
+    },
+    actions: {
+        description: `User-defined actions.\n\nIt can be either a Node or a function using the children-as-function pattern to pass a close function for use anywhere within the actions. This provides a lot of flexibility in terms of what actions may trigger the Popover to close the popover dialog`,
+        type: {
+            summary: "React.Node | (({|close: () => mixed|}) => React.Node)",
+        },
+        options: (Object.keys(ActionsMappings): Array<React.Node>),
+        mapping: ActionsMappings,
+    },
+    closeButtonVisible: {
+        description: `When true, the close button is shown; otherwise, the close button is not shown.`,
+        table: {
+            type: {
+                summary: "boolean",
+            },
+        },
+    },
+    closeButtonLabel: {
+        description: "Close button label for use in screen readers",
+        control: {type: "text"},
+    },
+    style: {
+        description: "Optional custom styles.",
+        control: {type: "object"},
+        table: {
+            category: "Styling",
+            type: {
+                summary: "StyleType",
+            },
+        },
+    },
+    testId: {
+        description: "Test ID used for e2e testing.",
+        control: {type: "text"},
+        table: {
+            type: {
+                summary: "string",
+            },
+        },
+    },
+    icon: {
```

Comment:
It looks like pressing the "Set Object" button on the `Icon` prop in the controls causes the site to go black and crash for me. Is this happening for you too?

Reply:
Okay, that was happening in the Docs tab. In the canvas tab, it gives me a "Minified React error #31" error.

Reply:
oh nice catch! I'll create some `mappings` to provide some valid examples.

Reply:
You can see the changes here: https://5e1bf4b385e3fb0020b7073c-wzaxfiexxw.chromatic.com/?path=/story/popover-popovercontent--default

Reply:
Great idea! FTR I added this in a separate PR... see: https://github.com/Khan/wonder-blocks/pull/1577/commits/b8bcee515c936c86d2e497d559c14c89d061fbd1